kong/api/routes/kong.lua

```diff
@@ -98,6 +100,9 @@ return {
         }
       end
 
+      local configuration = kong.configuration.remove_sensitive()
+      configuration.log_level = LOG_LEVELS[get_sys_filter_level()]
```

Comment:
not sure if it's a good idea to change this, as then it doesn't accuratly represent
what's in kong.conf. if we really want to see it in `:8001/`, i would suggest to
put it at top level

Reply:
if we don't change this, it won't accurately represent the dynamically changed (current) log level, it only show the kong.conf (old) log level

Reply:
@fffonion nonetheless, could you please add this suggestion to the Design Doc? we have agreed to change the `:8001/` endpoint to show current, up-to-date information about the log level considering the ones changed by the API proposed in this PR. So if we change this, we need to also discuss there.

Reply:
This is fine, in stead of showing
```
{
  configuration: {
    log_level: notice,
  }
}
```
i'm suggesting to return something like 
```
{
  configuration: {
    log_level: notice,
  },
  log_level: debug, # maybe dynamic_log_level ? whatever makes clearer to you
}
```

Reply:
Thanks for adding that to the Design Doc. Let's discuss there.